web/models/user.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MovieWagerApi.User do
+  use MovieWagerApi.Web, :model
+
+  schema "users" do
+    field :screen_name, :string
+    field :profile_image_url, :string
+    field :twitter_id, :integer
+    field :name, :string
```

Comment:
bleh, I was also hesitant about storing first and last name in `name` but I now realize you did it to make it easier to work with Twitter. I have a similar question as @michaelpelletier, is Twitter the only way for a user to register/authenticate? Setting these fields on the user table may make life harder should you wish to add FB or traditional email/password registration.